wolfcrypt/src/asn.c

```diff
@@ -14824,21 +14833,29 @@ int ExtractDate(const unsigned char* date, unsigned char format,
     certTime->tm_hour = tm_hour;
     certTime->tm_min  = tm_min;
     certTime->tm_sec  = tm_sec;
+    /* end AVR */
 #else
-    /* adjust tm_year, tm_mon */
-#ifdef WOLFSSL_LINUXKM
-    if (GetTime_Long(&certTime->tm_year, date, idx) != 0) return 0;
-#else
-    if (GetTime(&certTime->tm_year, date, idx) != 0) return 0;
-#endif
+    /* Everything else: not AVR */
```

Comment:
Just do `#else /* AVR */`

Reply:
got it. although it was `#else /* !AVR */`  (the section above was for AVR) ... assuming the `else` is for what follows.